docs/glossary.md

```diff
@@ -302,10 +320,73 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
 #### [Namespace](/namespaces)
 
-A Namespace is a unit of isolation within the Temporal Platform
+A Namespace is a unit of isolation within the Temporal Platform.
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+#### [Nexus API Definitions](/nexus#nexus-api-defintions)
+
+A common code package, schema, or documentation that a Caller can use to obtain Service and Operation names as associated input/output types a Service will accept for a given Operation.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus API Registry](/nexus#nexus-api-registry)
+
+An account-scoped list of Endpoints and Services available for use within a Temporal Cloud account, including an Endpoint description of the Services and Operations which the Endpoint supports. 
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Client](/nexus#nexus-client)
+
+Used by the Caller to connect to a Nexus Endpoint and Nexus Service.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Endpoint](/nexus#nexus-endpoint)
+
+A Nexus Endpoint is like an API proxy that can serve one or more Nexus Services from a target such as a Worker listening on a target Temporal Namespace and target Task Queue. 
```

Comment:
Curious (possibly stupid question) -- what makes it "like" an API proxy rather than just "an API proxy"?

Reply:
Would be fine removing "like" if we're comfortable with "API proxy" in this context translating to a  Temporal Nexus API proxy. Calling it an API proxy does make the concept immediately understandable to most everyone we have talked with.

Why "like"? Most traditional API proxies are typical HTTP request/response, whereas a Temporal Nexus Endpoint routes via Temporal's queue-based worker architecture via a namespace + task queue (and some Nexus RPC internally across some hops), so it didn't feel correct to position it as a traditional API proxy, but also it does perform the functions of an API proxy, perhaps just with a Temporal transport for part of the journey from caller to handler.

Also note that a Temporal Nexus Endpoint is more than just the low-level [Nexus endpoint](https://github.com/nexus-rpc/api/blob/30ebcf5ded186ccfc70759f864ce8a43d6c73859/SPEC.md?plain=1#L17), which does not route to Temporal namespaces, and Nexus RPC is agnostic to Temporal entirely -- much like a [Google Cloud Endpoint](https://cloud.google.com/endpoints) is a gRPC endpoint, but is also doing their own API proxy/gateway things, and gRPC is agnostic to Google Cloud Endpoints.

Reply:
Thanks for the context. Going to removing the word "like" for now unless there's an alternative.